libpod/runtime.go

```diff
@@ -533,6 +537,11 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 		}
 		runtime.config.Network.NetworkBackend = string(netBackend)
 		runtime.network = netInterface
+
+		// Using sync once value to only init the store exactly once and only when it will be actually be used.
+		runtime.ArtifactStore = sync.OnceValues(func() (*artStore.ArtifactStore, error) {
```

Comment:
Any reason this is a `sync.OnceValues()` instead of just calling NewArtifactStore every time? I know if won't be used every time, but it doesn't feel like that alone is a reason to complicate this

Reply:
I have left OnceValues() in but if you want me to change it I can drop it and always init the the store there.

Reply:
I am not committed to changing it, so I'll resolve the conversation